recipes/opencv-python-headless/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "opencv-python-headless" %}
+{% set version = "4.5.3.56" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+build:
+  noarch: generic
+  number: 0
+requirements:
+  host:
+    - python =3.8
```

Comment:
why is the pin to 3.8 needed?

Reply:
this is the part I'm not certain. this is a dependency of cellpose recipe #30492. 

When I test this recipe locally with cellpose recipe, without the pin, `conda install opencv-python-headless --use-local` installed the package under python3.9/. thus caused an error in the cellpose recipe build.

then I changed this recipe, added the pin, and re-test it locally, then cellpose recipe worked.

thanks for the help